src/game_picture.cpp

```diff
@@ -186,37 +186,37 @@ void Game_Picture::Move(const MoveParams& params) {
 	// Note that data.effect_mode doesn't necessarily reflect the
 	// last effect set. Possible states are:
 	//
-	// * effect_mode == 0 && finish_effect == 0
+	// * effect_mode == 0 && finish_effect_power == 0
```

Comment:
Still using magic numbers here, maybe update/rewrite the comment

Reply:
Fixed